core/client/pom.xml

```diff
@@ -138,12 +138,7 @@
         <dependency>
           <groupId>org.apache.curator</groupId>
           <artifactId>curator-framework</artifactId>
-          <exclusions>
-            <exclusion>
-              <groupId>org.jboss.netty</groupId>
-              <artifactId>netty</artifactId>
-            </exclusion>
-          </exclusions>
+          <type>provided</type>
```

Comment:
as talked offline, make sure flink runtime has `curator-framework` on classpath. 
Also update the comment